test/functional/api/cas/cli_help_messages.py

```diff
@@ -0,0 +1,276 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+casadm_help = [
+    r"Cache Acceleration Software Linux",
+    r"Usage: casadm \<command\> \[option\.\.\.\]",
```

Comment:
I'm not sure if you need to (or even can) escape characters with backslash in raw strings. If you tested it and it's ok, then I'm backing off. ;) The rest looks good to me.

Reply:
Yeah, it is required by regex.